log-processing/jobs/query-logs.yaml

```diff
@@ -1,31 +1,19 @@
-Name: Query logs
+Name: Query logs - 5xx status codes
```

Comment:
lets discuss the queries we want to ship here. In the original post, ops job accepted the query as variable, but had fixed basic and deep-dive batch queries